airflow/providers/facebook/ads/hooks/ads.py

```diff
@@ -92,7 +92,7 @@ def facebook_ads_config(self) -> Dict:
         """
         self.log.info("Fetching fb connection: %s", self.facebook_conn_id)
         conn = self.get_connection(self.facebook_conn_id)
-        config = conn.extra_dejson
+        config = {**conn.extra_dejson, "access_token": conn.password}
```

Comment:
Should there be some warning/message to the user noting that they should set `access_token` in the Connection's `password` attr rather than in Extras as well?

Side note, this hook desperately needs some documentation re: setting up a connection.

Reply:
Yeah, _or_ (if we decide setting the token in extras is acceptable), there should be some checks to make sure the empty password does not override the token in extras.

Reply:
I don't think it's appropriate for the access_token to be passed in the extra parameter since it is essentially the password for the Facebook API and should not be exposed. +1 need for documentation.